pycobertura/reporters.py

```diff
@@ -13,24 +13,40 @@
 env.filters["line_reason"] = filters.line_reason_icon
 env.filters["is_not_equal_to_dash"] = filters.is_not_equal_to_dash
 env.filters["misses_color"] = filters.misses_color
+env.filters[
+    "get_max_number_of_values_in_dict"
+] = filters.get_max_number_of_values_in_dict
 
 headers_with_missing = ["Filename", "Stmts", "Miss", "Cover", "Missing"]
 headers_without_missing = ["Filename", "Stmts", "Miss", "Cover"]
+headers_hideable = headers_without_missing.remove("Filename")
 
 
 class Reporter:
-    def __init__(self, cobertura, ignore_regex=None):
+    def __init__(self, cobertura, ignore_regex=None, hide_columns=[]):
         self.cobertura = cobertura
         self.ignore_regex = ignore_regex
+        self.sep = "" if "," in hide_columns else ","
```

Comment:
FYI @aconrad Needed to do this because command line passes `['S','t','m','t','s',',','M','i','s','s','i','n','g']` whereas runner.invoke passes `["Stmts","Missing"]`

Reply:
outdate.